readthedocs/doc_builder/python_environments.py

```diff
@@ -219,27 +219,15 @@ def install_core_requirements(self):
         else:
             requirements.extend(
                 [
-                    self.project.get_feature_value(
-                        Feature.USE_SPHINX_LATEST,
-                        positive="sphinx",
-                        negative="sphinx<2",
-                    ),
-                    # If defaulting to Sphinx 2+, we need to push the latest theme
-                    # release as well. `<0.5.0` is not compatible with Sphinx 2+
-                    self.project.get_feature_value(
-                        Feature.USE_SPHINX_LATEST,
-                        positive="sphinx-rtd-theme",
-                        negative="sphinx-rtd-theme<0.5",
-                    ),
+                    "sphinx",
+                    "sphinx-rtd-theme",
                     self.project.get_feature_value(
                         Feature.USE_SPHINX_RTD_EXT_LATEST,
                         positive="readthedocs-sphinx-ext",
                         negative="readthedocs-sphinx-ext<2.3",
                     ),
                 ]
             )
-            if not self.project.has_feature(Feature.USE_SPHINX_LATEST):
-                requirements.extend(["jinja2<3.1.0"])
```

Comment:
Yes please, this one is super weird... I don't know what implications that it may or may not have had. But we removed this requirement from sphinx-rtd-theme so it quite clearly should also be removed here. Pretty sure it's related to supporting some very old version of Sphinx.